src/gc/gc.h

```diff
@@ -534,8 +257,8 @@ class GCHeap {
     virtual void TemporaryDisableConcurrentGC() = 0;
```

Comment:
All these abstract methods that are defined on `IGCHeap` should be not be defined again on `IGCHeapInternal`. It makes it hard to read figure out is actually on `IGCHeapInternal`.